.github/workflows/triggered.yml

```diff
@@ -19,23 +19,23 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest"]
-        python-version: ["3.7", "3.10", "pypy-3.8", "pypy-3.9"]
+        python-version: ["3.8", "3.12", "pypy-3.8", "pypy-3.9"]
```

Comment:
Please drop pypy-3.8, it is really not supported. Also pypy3.9 has been dropped from support, only pypy3.10 is currently supported.